Core.ApplicationServices/Organizations/IOrganizationService.cs

```diff
@@ -45,5 +45,6 @@ public interface IOrganizationService
         Maybe<OperationError> RemoveOrganization(Guid organizationUuid, bool enforceDeletion);
 
         Result<IEnumerable<Organization>, OperationError> GetUserOrganizations(int userId);
+        bool CanActiveUserModifyCvr();
```

Comment:
you are missing the org id here.. I know the implementation does not care - today, but that could change tomorrow, so keep the interface good and it will be better suited for the future... that also changes the result to `Result<bool,OperationError>` since the client may ask in an org that:
- Does not exist
- Which they have no permission to access and thereby no permission to ask for modification access in